pytket/docs/changelog.rst

```diff
@@ -14,6 +14,7 @@ Fixes:
 * Fix `symbol_substitution` not preserving opgroups.
 * Remove hardware inefficient circuit construction in `_tk1_to_rzsx`
 * Support converting conditional `RangePredicate`s to QASM.
+* Fix `maxwidth` parameter of `circuit_from_qasm_str`
```

Comment:
Also mention the register-resizing fix here?

Reply:
Not sure what you mean, do you want me to add more details on how the fix is done?

Reply:
We're also adding the `scratch_reg_resize_pass()` to the QASM conversion, might be worth mentioning this.

Reply:
done in [4488ded](https://github.com/CQCL/tket/pull/1653/commits/4488ded1b194ddf148a059e3040f73efe4b877ba)